Makefile

```diff
@@ -18,3 +20,34 @@ runDebug: build
 clean:
 		go clean
 		rm -rf ${BINARY_NAME}
+
+# formatting and linting
+fmt:
+	gofmt -s -w .
+# change to `run ./pkg/*` after agents are implemented
+# should just be `run`, but seems to be problems with go.work  
+# nb: using cd to /pkg/infra dosen't fix this (on wsl2)
+check:
+	golangci-lint run ./pkg/infra\
+		--timeout=5m0s \
+		--enable=gofmt \
+		--enable=unconvert \
+		--enable=unparam \
+		--enable=asciicheck \
+		--enable=misspell \
+		--enable=decorder \
+		--enable=depguard \
+		--enable=nilerr \
+		--enable=exportloopref \
+		--enable=whitespace \
+		--enable=goimports
```

Comment:
Could probably change this in the future to run with the [.golangci.yaml](.golangci.yaml) config file

Reply:
ur right, I've updated Makefile